scripts/rspec/rspec-templates/CommonBaseClassTemplate.cs

```diff
@@ -30,4 +30,4 @@ public abstract class $DiagnosticClassName$Base : SonarDiagnosticAnalyzer
         protected const string DiagnosticId = "$DiagnosticId$";
         protected const string MessageFormat = "";
     }
-}
+}
```

Comment:
Let's add more stuff into this class. When base class is present, it should have the `SupportedDiagnostics`, `rule` and constructor with resource manager argument.